browser/permissions/permission_manager_browsertest.cc

```diff
@@ -192,36 +190,39 @@ IN_PROC_BROWSER_TEST_F(PermissionManagerBrowserTest, RequestPermissions) {
 
   auto* permission_request_manager = GetPermissionRequestManager();
   EXPECT_FALSE(permission_request_manager->IsRequestInProgress());
-  struct {
+  struct TestEntries {
     std::vector<std::string> addresses;
     ContentSettingsType type;
     blink::PermissionType permission;
-  } cases[] = {{{"0xaf5Ad1E10926C0Ee4af4eDAC61DD60E853753f8A",
-                 "0xaf5Ad1E10926C0Ee4af4eDAC61DD60E853753f8B"},
-                ContentSettingsType::BRAVE_ETHEREUM,
-                blink::PermissionType::BRAVE_ETHEREUM},
-               {{"BrG44HdsEhzapvs8bEqzvkq4egwevS3fRE6ze2ENo6S8",
-                 "JDqrvDz8d8tFCADashbUKQDKfJZFobNy13ugN65t1wvV"},
-                ContentSettingsType::BRAVE_SOLANA,
-                blink::PermissionType::BRAVE_SOLANA}};
-  for (size_t i = 0; i < sizeof(cases) / sizeof(cases[0]); ++i) {
-    const std::vector<std::string>& addresses = cases[i].addresses;
-    RequestType request_type = ContentSettingsTypeToRequestType(cases[i].type);
-    EXPECT_TRUE(IsPendingGroupedRequestsEmpty(cases[i].type)) << "case: " << i;
-
-    std::vector<blink::PermissionType> permissions(addresses.size(),
-                                                   cases[i].permission);
-    std::vector<url::Origin> sub_request_origins(addresses.size());
-    for (size_t j = 0; j < addresses.size(); ++j) {
+  };
+  auto entries = std::to_array<TestEntries>(
+      {{{"0xaf5Ad1E10926C0Ee4af4eDAC61DD60E853753f8A",
+         "0xaf5Ad1E10926C0Ee4af4eDAC61DD60E853753f8B"},
+        ContentSettingsType::BRAVE_ETHEREUM,
+        blink::PermissionType::BRAVE_ETHEREUM},
+       {{"BrG44HdsEhzapvs8bEqzvkq4egwevS3fRE6ze2ENo6S8",
+         "JDqrvDz8d8tFCADashbUKQDKfJZFobNy13ugN65t1wvV"},
+        ContentSettingsType::BRAVE_SOLANA,
+        blink::PermissionType::BRAVE_SOLANA}});
+  for (const auto& entry : entries) {
+    RequestType request_type = ContentSettingsTypeToRequestType(entry.type);
+    EXPECT_TRUE(IsPendingGroupedRequestsEmpty(entry.type))
+        << "case: " << entry.type;
```

Comment:
I'd suggest SCOPED_TRACE() instead of `<< "case: " << entry.type;` copypastes

Reply:
Fixed. Didn't know about this one. Thanks.